src/component/book/location/BookShelfSVG.tsx

```diff
@@ -0,0 +1,227 @@
+        }`}
+        d="M281 0V97.5H223V132V198L141 196L0 0H281Z"
+        fill="#2FC949"
+      />
+      <path
+        key="O"
+        className={`shelf-3__callSign-O callSignPath ${
+          highlight === "O" ? "highlight_svg" : ""
+        }`}
+        d="M143.5 197.5L231 323L176.5 323L176.5 394.492L0.499925 394.492L0.499923 -0.000356686L143.5 197.5Z"
+        fill="#32BBD6"
+      />
+      <path
+        key="H"
+        className={`shelf-3__callSign-H callSignPath ${
+          highlight === "H" ? "highlight_svg" : ""
+        }`}
+        d="M47.9995 166.999L0.500011 99.0001L81.4997 101.189L81.4997 0.49883L140 0.498843L140 194.5L141.5 297.999L34.4995 297.999L34.4996 225.999L88.5 225.999L47.9995 166.999Z"
+        fill="#A06565"
+        transform="translate(142, 97)"
+      />
+    </svg>
+  );
+};
+
+const ShelfFour = () => {
+  return (
+    <div className="shelffour">
+      <div></div>
+      {/* <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="201"
+        height="183"
+        viewBox="0 0 201 183"
+        fill="none"
+      >
+        <path d="M0 0H200V183H0V0Z" fill="#D9D9D9" />
+      </svg> */}
+    </div>
+  );
+};
+
+const ShelfFive = ({ highlight }: { highlight: string }) => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="201"
+      height="183"
+      viewBox="0 0 201 183"
+      fill="none"
+    >
+      {/* <path d="M0 0H200V183H0V0Z" fill="#D9D9D9" /> */}
+      <path
+        key="L"
+        className={`shelf-5__callSign-L callSignPath ${
+          highlight === "L" ? "highlight_svg" : ""
+        }`}
+        d="M1 0H201V45H1V0Z"
+        fill="#9AC59E"
+      />
+      <path
+        key="C"
+        className={`shelf-5__callSign-C callSignPath ${
+          highlight === "C" ? "highlight_svg" : ""
+        }`}
+        d="M1 45H201V93.4375H74.3906V138H37.4807H1V45Z"
+        fill="#D877C4"
+      />
+      <rect
+        key="a"
+        className={`shelf-5__callSign-a callSignPath ${
+          highlight === "a" ? "highlight_svg" : ""
+        }`}
+        x="1"
+        y="138"
+        width="200"
+        height="45"
+        fill="#70A4C7"
+      />
+      <path
+        key="M"
+        className={`shelf-5__callSign-M callSignPath ${
+          highlight === "M" ? "highlight_svg" : ""
+        }`}
+        d="M74 93H201V138H74V93Z"
+        fill="#B3B3B3"
+      />
+    </svg>
+  );
+};
+
+const BookShelfSVG = ({ shelfIndex }: BookShelfSVGProps) => {
+  const { callSignFirstChar } = useBookContext();
+  const renderShelf = () => {
+    switch (shelfIndex) {
+      case 0:
+        return <ShelfOne highlight={callSignFirstChar} />;
+      case 1:
+        return <ShelfTwo highlight={callSignFirstChar} />;
+      case 2:
+        return <ShelfThree highlight={callSignFirstChar} />;
+      case 3:
+        return <ShelfFour />;
+      case 4:
+        return <ShelfFive highlight={callSignFirstChar} />;
+      default:
+        return <ShelfFour />;
```

Comment:
```suggestion
        return <ShelfFour />;
```
```suggestion
        return <ShelfError />;
```

이런식으로 `default`는 예외처리로 빼면 더 좋을거 같아요